### DISCUSSION TITLE
Reading LIS grib2 files in Python

### DISCUSSION THREAD

**Discussion by cbblanke** [NASA-LIS/LISF#1620] [Q&A] [answered]
Does anyone have an example of how to read the grib output in Python?

**Comment by dmocko** [comment]
Hi @cbblanke 

Perhaps @emkemp @karsenau or @smahanam can help you with this.

**Reply by emkemp** [reply, marked as answer]
@cbblanke I use eccodes.

I have an old script for processing GRIB2 output from LVT (customized for particular variables I was interested in).  On Discover, look in /discover/nobackup/projects/usaf_lis/emkemp/AFWA/python/plot_lis72_24hr_grib2.py.

Some documentation/examples are available at https://confluence.ecmwf.int/display/ECC/GRIB+examples.
